app/controllers/hq/sponsors_controller.rb

```diff
@@ -47,6 +48,12 @@ def load_sponsor
     @sponsor= Sponsor.find(params[:id])
   end
 
+  def load_active_and_inactive_sponsorships
+    sponsorships = Sponsorship.where("sponsor_id = ?", @sponsor.id).eager_load(:orphan, :sponsor).all
```

Comment:
Is it necessary to eager_load sponsor here considering that the sponsor is already in memory as `@sponsor`? Also, what do you think about defining a default scope on Sponsorship that includes orphans & sponsors tables, the way @PurityControl has done for Orphan?